packages/core/src/external/aws/s3.ts

```diff
@@ -217,30 +217,69 @@ export class S3Utils {
 
     return newKey;
   }
-  async uploadFile(
-    bucket: string,
-    key: string,
-    file: Buffer
-  ): Promise<AWS.S3.ManagedUpload.SendData> {
+  async uploadFile({
+    bucket,
+    key,
+    file,
+    contentType,
+  }: {
+    bucket: string;
+    key: string;
+    file: Buffer;
+    contentType?: string;
+  }): Promise<AWS.S3.ManagedUpload.SendData> {
     return new Promise((resolve, reject) => {
-      this._s3.upload(
-        {
-          Bucket: bucket,
-          Key: key,
-          Body: file,
-        },
-        (err, data) => {
-          if (err) {
-            console.error("Error during upload:", err);
-            reject(err);
-          } else {
-            console.log("Upload successful");
```

Comment:
this was removed, but not added back, intentional?

Reply:
You mean the specific log line: 
`console.log("Upload successful");` ? Will add back